e2e/tests/adminMangesHearings_test.js

```diff
@@ -254,11 +254,11 @@ Scenario('HMCTS admin cancels and re-lists hearing', async ({I, caseViewPage, ma
 });
 
 Scenario('HMCTS admin adds past hearing', async ({I, caseViewPage, manageHearingsEventPage}) => {
-  hearingStartDate = moment().subtract(5,'m').toDate();
-  hearingEndDate = moment(hearingStartDate).subtract(5,'m').toDate();
+  hearingStartDate = moment().subtract(10,'m').toDate();
```

Comment:
what was wrong the orginal -5mins?

Reply:
nothing wrong with the original `- 5 minutes`. only hearing end date needed change as it was earlier than the hearing start date and the test was failing with the new validation error.